postgres-appliance/Dockerfile

```diff
@@ -123,6 +124,7 @@ RUN export DEBIAN_FRONTEND=noninteractive \
 \
     && curl -sL https://github.com/CyberDem0n/bg_mon/archive/$BG_MON_COMMIT.tar.gz | tar xz \
     && curl -sL https://github.com/RafiaSabih/pg_auth_mon/archive/$PG_AUTH_MON_COMMIT.tar.gz | tar xz \
+    && curl -sL https://github.com/RafiaSabih/pg_mon/archive/$PG_MON_COMMIT.tar.gz | tar xz \
```

Comment:
Lets move it [here](https://github.com/zalando/spilo/pull/420/files#diff-82f985fcf1262fb2d3d1843b9e74d9c3R109)